www/js/survey/enketo/answer.js

```diff
@@ -52,6 +52,9 @@ angular.module('emission.survey.enketo.answer', [
     UserProfileSurvey: (xmlDoc) => {
       return 'Answered';
     },
+    TimeUseSurvey: (xmlDoc) => {
+      return 'Answered';
+    }
```

Comment:
might want to check with @asiripanich on whether we want to pull out some fields to display in the label for this as well (since it will show up in the diary).